bin/cache_repo

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+AWS_BUCKET=$1
+
+DATE=$(date +"%Y-%m-%d")
+
+mkdir -p /tmp/$DATE
```

Comment:
```suggestion
mkdir -p "/tmp/$DATE"
```

For consistency with the quoted usage below. Or, vice versa we could remove the quotes for the usage below.

IMHO, better to err on the side of caution and quote everything.

Reply:
> better to err on the side of caution and quote everything

Yeah, it _should_ be impossible for this to break, but better safe than sorry 🙂